src/composables/useNetwork.ts

```diff
@@ -1,18 +1,30 @@
-import { Network } from '@balancer-labs/sdk';
 import { computed, ref } from 'vue';
 
 import config from '@/lib/config';
 import { configService } from '@/services/config/config.service';
+import { Network } from '@balancer-labs/sdk';
 
 /**
  * STATE
  */
-const DEFAULT_NETWORK_ID =
-  process.env.VUE_APP_NETWORK != null
-    ? (Number(process.env.VUE_APP_NETWORK) as Network)
-    : Network.MAINNET;
+const windowAvailable = typeof window !== 'undefined';
+const localStorageNetworkId: Network | undefined =
+  windowAvailable && localStorage.getItem('networkId')
+    ? (Number(localStorage.getItem('networkId')) as Network)
+    : undefined;
+
+const NETWORK_ID =
```

Comment:
Are you sure? This scenario happens in pretty much only case - when user opens `/` route without a network selected in url. Should we not show Ethereum by default to highlight our main pools like like 80-20 BAT-ETH and Aave boosted?

Reply:
imo I don't think we should be detecting the user's wallet's network and auto redirecting. I think that could be especially confusing if they follow a link to a pool on a specific network, for example: A user who's wallet is connected to mainnet clicks on an external URL linking to a polygon pool, on landing on the page we detect their network is mainnet and throw them back to the home page.

Reply:
@garethfuller The users wallet would only come after the URL in priority. Priority should be URL flag -> local storage network -> users wallet currently selected network -> mainnet. 

@Matthews3301 A lot of people only use Polygon (or L2's) avoid mainnet because it's too expensive for them. It would make more sense for them to just be shown the Polygon homepage as their default landing page. 

It's also useful for repeat Balancer users - so that they don't have to continually select their favorite network every time they open Balancer. Though maybe this is covered by localStorageNetworkId.

Reply:
I still don't agree that we should auto-redirect based on a user's wallet (except in the case that we talked about Matthew, where a user is already connected to the correct network and then switches).

I do agree that we should store their last used network in localStorage and use that, but I'm not sure how. One issue for example, similar to what I described above: what if the user last used polygon and then left Balancer, they then click on a link to a mainnet pool page or to the arbitrum pools list from an external app, do we redirect them? We shouldn't. So in what cases would we use the localStorage variable to redirect on landing?

Reply:
I think the localStorage network or using users wallet network only makes sense on the main pools list or trade page without any network in the URL. Which seems to be the logic here, having the network in the URL overrides all.